docs/intro/go.md

```diff
@@ -0,0 +1,107 @@
+---
+order: 2
+description: Install the Go programming language
+---
+
+# Install Go
+
+In this tutorial, you will install the Go programming language (golang) on your local computer. Follow the instructions for your operating system:
+
+## macOS
+
+* Download the latest MacOS installer package from <https://golang.org/dl/>.
+* Open the downloaded package and follow the prompts through to completion.
+* By default, the package installs the Go distribution to `/usr/local/go`, however it is always best to define the path explicitly:
+
+    1. Open or create a `~/.bash_profile` file with your favorite command-line text editor.
+    2. Add the following lines:
+
+        ```sh
+        export GOPATH=$HOME/go
```

Comment:
```suggestion
```

go path is no longer required.

Reply:
It might be useful if you run `starport c build` and then want to use the chain's binary.

Reply:
In which, we should use `export PATH=$PATH:$(go env GOPATH)/bin`.